### PR TITLE
[TASK] Add images organization

### PR DESCRIPTION
**Pull request by Hasan-Shaker**

### CODE REVIEW

**Review by xperseguers — COMMENTED** [review]
`[TASK] Add images organization` doesn't really mean anything in English, better would be `[TASK] Add images for organization` or `[FEATURE] Add images for organization`

File: Configuration/TCA/Overrides/tx_staffdirectory_domain_model_organization.php [hunk 1]
```diff
@@ -24,7 +24,7 @@
     ],
     'tx_staffdirectoryorganization_locality' => [
         'exclude' => false,
-        'label' => 'LLL:EXT:staffdirectory_organization/Resources/Private/Language/locallang_db.xlf:tx_staffdirectory_domain_model_organization.locality',
+        'label' => 'LLL:EXT:core/Resources/Private/Language/locallang_general.xlf:LGL.locality',
```
Comment (xperseguers) [inline Configuration/TCA/Overrides/tx_staffdirectory_domain_model_organization.php:27]: This has nothing to do with the PR

File: Configuration/TCA/Overrides/tx_staffdirectory_domain_model_organization.php [hunk 2]
```diff
@@ -61,9 +61,48 @@
...
+                            ],
+                        ],
+                    ],
+                ],
+                $GLOBALS['TYPO3_CONF_VARS']['GFX']['imagefile_ext']
+            ),
+        ],
+    ]
+);
+
```
Comment (xperseguers) [inline Configuration/TCA/Overrides/tx_staffdirectory_domain_model_organization.php:97]: Why don't you add this to `$tempColumns` instead?

File: Configuration/TCA/Overrides/tx_staffdirectory_domain_model_organization.php [hunk 3]
```diff
@@ -61,9 +61,48 @@
...
+\TYPO3\CMS\Core\Utility\ExtensionManagementUtility::addToAllTCAtypes(
+    'tx_staffdirectory_domain_model_organization',
+    'LLL:EXT:core/Resources/Private/Language/locallang_general.xlf:LGL.images,
+        tx_staffdirectoryorganization_image,',
```
Comment (xperseguers) [inline Configuration/TCA/Overrides/tx_staffdirectory_domain_model_organization.php:101]: The trailing comma is certainly useless here

File: Resources/Private/Language/locallang_db.xlf [hunk 4]
```diff
@@ -3,9 +3,6 @@
 	<file source-language="en" datatype="plaintext" date="2024-03-21T16:51:33Z">
 		<header/>
 		<body>
-			<trans-unit id="tx_staffdirectory_domain_model_organization.locality">
```
Comment (xperseguers) [inline Resources/Private/Language/locallang_db.xlf:6]: This has nothing to do with the PR

**Review by xperseguers — CHANGES_REQUESTED** [review]

File: Configuration/TCA/Overrides/tx_staffdirectory_domain_model_organization.php (hunk 2 above)
Comment (Hasan-Shaker) [reply Configuration/TCA/Overrides/tx_staffdirectory_domain_model_organization.php:97]: I wanted to put the image tab not in contact tab.

File: Resources/Private/Language/locallang_db.xlf (hunk 4 above)
Comment (Hasan-Shaker) [reply Resources/Private/Language/locallang_db.xlf:6]: this is a just correction.

File: Configuration/TCA/Overrides/tx_staffdirectory_domain_model_organization.php (hunk 1 above)
Comment (Hasan-Shaker) [reply Configuration/TCA/Overrides/tx_staffdirectory_domain_model_organization.php:27]: this is a just correction.

**Comment by Hasan-Shaker** [comment]
OK, I'll revisit my commit

**Pull request closed without merging**